usersession/agent/session_agent.go

```diff
@@ -180,6 +182,10 @@ func (s *SessionAgent) Init() error {
 	if err != nil {
 		return err
 	}
+
+	// Set up notification manager
+	s.notificationMgr = notification.NewNotificationManager(s.bus, "io.snapcraft.SessionAgent")
```

Comment:
Note that `s.bus` may be nil here (see comment in `tryConnectSessionBus`). It's really only like that due to Ubuntu 16.04's weird mix of systemd and Upstart, but it is a case we tried to cover. Not sure what is best to do now.